spec/factories/providers.rb

```diff
@@ -52,6 +53,8 @@
     end
 
     after(:create) do |provider, evaluator|
+      next if evaluator.skip_associated_data
```

Comment:
`after` blocks aren't _just_ for creating associated data ... what if we just added `sites: [], enrichments: []`, etc, to the `create(:provider, ...)` below? Also, less machinery to add here.

Reply:
Done